arex-web-core/src/main/java/com/arextest/web/core/repository/mongo/AppContractRepositoryImpl.java

```diff
@@ -47,12 +57,83 @@ public boolean saveAppContractList(List<AppContractDto> applicationInfoDtos) {
             }
             bulkOperations.execute();
         } catch (Exception e) {
-            LogUtils.error(LOGGER, "saveApplicationList failed! list:{}", applicationInfoDtos, e);
+            LogUtils.error(LOGGER, "saveApplicationList failed! list:{}", appContractDtos, e);
             return false;
         }
         return true;
     }
 
+    @Override
+    public List<AppContractDto> upsertAppContractListWithResult(List<AppContractDto> appContractDtos) {
```

Comment:
There should only be codes for database operations, business processing logic should not be in this layer

Reply:
reasonable